packages/alert-banner/src/AlertBanner.ts

```diff
@@ -0,0 +1,146 @@
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    public open = false;
+
+    @property({ type: Boolean, reflect: true })
+    public dismissible = false;
+
+    /**
+     * The variant applies specific styling when set to `negative` or `info`;
+     * `variant` attribute is removed when it's passed an invalid variant.
+     *
+     * @param {String} variant
+     */
+    @property({ type: String })
+    public set variant(variant: AlertBannerVariants) {
+        if (variant === this.variant) {
+            return;
+        }
+        const oldValue = this.variant;
+        if (this._isValidVariant(variant)) {
+            this.setAttribute('variant', variant);
+            this._variant = variant;
+        } else {
+            this.removeAttribute('variant');
+            this._variant = '';
+        }
+        this.requestUpdate('variant', oldValue);
+    }
+
+    public get variant(): AlertBannerVariants {
+        return this._variant;
+    }
+
+    private _variant: AlertBannerVariants = '';
+
+    protected _isValidVariant(variant: string): boolean {
+        return VALID_VARIANTS.includes(variant);
+    }
+
+    protected renderIcon(variant: string): TemplateResult {
+        switch (variant) {
+            case 'info':
+                return html`
+                    <sp-icon-info
+                        label="Information"
+                        class="type"
+                    ></sp-icon-info>
+                `;
+            case 'negative':
+                return html`
+                    <sp-icon-alert label="Error" class="type"></sp-icon-alert>
+                `;
+            default:
+                return html``;
+        }
+    }
+
+    private _shouldClose(): void {
+        const applyDefault = this.dispatchEvent(
+            new CustomEvent('close', {
+                composed: true,
+                bubbles: true,
+                cancelable: true,
+            })
+        );
+        if (applyDefault) {
+            this.close();
+        }
+    }
+
+    public close(): void {
+        this.open = false;
+    }
+
+    private _handleKeydown(event: KeyboardEvent): void {
+        if (event.code === 'Escape' && this.dismissible) {
+            this._shouldClose();
+        }
+    }
+
+    protected override render(): TemplateResult {
+        return html`
+            <div class="body" role="alert">
+                <div class="content">
+                    ${this.renderIcon(this.variant)}
+                    <div class="text"><slot></slot></div>
+                </div>
+                <slot name="action"></slot>
+            </div>
+            <div class="end">
+                ${this.dismissible
+                    ? html`
+                          <sp-close-button
+                              @click=${this._shouldClose}
+                              label="Close"
+                              static="white"
```

Comment:
is static="white" as per the design? Did you get a chance to test this with dark/white theme and also in HCM mode?

Reply:
Yes, that's part of the design and it's the same for toast. I've tested this for light/dark themes and also high-contrast. And I see we also have [screenshots](https://31876a81d3152f0d5cb17123fd5fc254--spectrum-web-components.netlify.app/review/) taken for high-contrast in our visual tests